ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml

```diff
@@ -465,6 +465,10 @@ spec:
           value: "$(params.pyxis_ssl_cert_secret_key)"
         - name: pyxis_ssl_key_secret_key
           value: "$(params.pyxis_ssl_key_secret_key)"
+      workspaces:
+        - name: source
+          workspace: pipeline
+          subPath: src
```

Comment:
I got an error trying to `oc apply` this because there isn't a `pipeline` workspace. You can probably use this instead:
```yaml
workspaces:
  - name: source
    workspace: repository
```

Let's also not dump them in the `src` directory so the git remains status clean. Can you please remove `subPath` from the others too?

Reply:
Done! Thanks :)